website/tables/aws/awscost_costexplorer_cost_current_month.md

```diff
@@ -0,0 +1,24 @@
+# Table: awscost_costexplorer_cost_current_month
+
+This table shows data for Awscost Costexplorer Cost Current Month.
```

Comment:
To fix this you should edit resources/plugin/plugin.go and add it to `awsExceptions`